Source/MLX/Cmlx+Util.swift

```diff
@@ -58,7 +58,31 @@ func mlx_map_values(_ mlx_map: mlx_map_string_to_array) -> [String: MLXArray] {
     return result
 }
 
-func new_mlx_map(_ dictionary: [String: MLXArray]) -> mlx_map_string_to_array {
+func mlx_map_string_values(_ mlx_map: mlx_map_string_to_string) -> [String: String] {
```

Comment:
similar to the `mlx_map_array_values()` function, this creates a `[String:String]` from `mlx_map_string_to_string`